backend/src/mailer/text/resetPassword.ts

```diff
@@ -0,0 +1,14 @@
+export const resetPassword = {
+  de: {
+    subject: 'Gradido: Passwort zurücksetzen',
+    text: (data: { link: string; firstName: string; lastName: string; email: string }): string =>
```

Comment:
can we not use a string replacement like sprint here and have this file without any javascript, just placeholder definitions?